EIPS/eip-5792.md

```diff
@@ -1,390 +1,267 @@
 ---
 eip: 5792
-title: Wallet Function Call API
-description: Adds JSON-RPC methods for sending multiple function calls from the user's wallet, and checking their status
-author: Moody Salem (@moodysalem)
+title: Wallet Call API
+description: Adds JSON-RPC methods for sending multiple calls from the user's wallet, and checking their status
+author: Moody Salem (@moodysalem), Lukas Rosario (@lukasrosario), Wilson Cusack (@wilsoncusack), Dror Tirosh (@drortirosh)
 discussions-to: https://ethereum-magicians.org/t/eip-5792-wallet-abstract-transaction-send-api/11374
-status: Stagnant
+status: Draft
 type: Standards Track
 category: Interface
 created: 2022-10-17
 ---
 
 ## Abstract
 
-Defines new JSON-RPC methods for dapps to send batches of function calls from a user's wallet, as well as check
-on the status of those calls. The new methods are more abstract in regard to the underlying transactions than the
-existing transaction sending APIs to allow for differences between wallet implementations, e.g.
-smart contract wallets utilizing [EIP-4337](./eip-4337.md) or EOA wallets that support bundled transactions
-via [EIP-3074](./eip-3074.md). Dapps may use this more abstract interface to support different kinds of wallets without
-additional effort, as well as provide a better UX for sending bundles of function calls (
-e.g. [EIP-20](./eip-20.md) `#approve` followed by a contract call).
+Defines new JSON-RPC methods for apps to send batches of calls from a user's wallet, as well as check on the status of those calls. The new methods have a simple core while enabling additional transaction features (e.g. sponsored transactions with a paymaster) via the introduction of capabilities. To support this, there is also a new method for querying for the capabilities a given wallet supports. The new methods are more abstract in regard to the underlying transactions than the existing transaction sending APIs to allow for differences between wallet implementations, e.g. smart accounts utilizing [ERC-4337](./eip-4337.md) or EOA wallets that support batched transactions via [EIP-3074](./eip-3074.md). Apps may use this more abstract interface to support different kinds of wallets without additional effort, as well as provide a better UX for sending batches of function calls (e.g. [ERC-20](./eip-20.md) `approve` followed by a contract call).
```

Comment:
Not perfect, but I think this would improve 

```suggestion
Defines new JSON-RPC methods which enable apps to ask a wallet to process a batch of onchain write calls and to check on the status of those calls. Calls can be specified to make use of various wallet-defined capabilities, and there is also a new RPC to enable apps to discover which capabilities a wallet supports. 
```